x/gov/keeper/deposit.go

```diff
@@ -60,11 +60,8 @@ func (keeper Keeper) IterateDeposits(ctx context.Context, proposalID uint64, cb
 // Activates voting period when appropriate and returns true in that case, else returns false.
 func (keeper Keeper) AddDeposit(ctx context.Context, proposalID uint64, depositorAddr sdk.AccAddress, depositAmount sdk.Coins) (bool, error) {
 	// Checks to see if proposal exists
-	proposal, err := keeper.GetProposal(ctx, proposalID)
+	proposal, err := keeper.Proposals.Get(ctx, proposalID)
 	if err != nil {
```

Comment:
Do you think it isn't necessary to do a check of `collections.ErrNotFound` for keeping the same behavior?

Reply:
I think we can use the `ErrNotFound` of collections which is more descriptive and standardized. This breaks consensus but it does not break state compatibility so I think it should be fine, it also allowed us to remove some errors from gov which I think are a little overkill. 

If you think this is not good I can revert the changes in a PR.